examples/VIAME_OcclusionSaliency.ipynb

```diff
@@ -47,10 +47,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!python -c \"import xaitk_saliency\" || pip install xaitk-saliency\n",
-    "!python -c \"import bioharn\" || pip install git+https://gitlab.kitware.com/viame/bioharn.git@dev/0.2.0  # test with 0.2.0\n",
-    "!python -c \"import torch\" || pip install \"torch==1.9.0\"\n",
-    "!python -c \"import torchvision\" || pip install \"torchvision==0.10.0\""
+    "!pip install -U pip\n",
+    "!pip install xaitk-saliency\n",
+    "!pip install \"torch==1.9.0\"\n",
+    "!pip install \"torchvision==0.10.0\"\n",
+    "!pip install git+https://gitlab.kitware.com/viame/bioharn.git@dev/0.2.0  # test with 0.2.0\n",
+    "# Remove opencv-python, which required libGL, which we don't require here, and replace with opencv-python-headless\n",
+    "!pip uninstall -y opencv-python opencv-python-headless  # make sure they're both gone.\n",
+    "!pip install opencv-python-headless"
```

Comment:
In the VIAME and scikit-learn notebooks, we simplified the first cell to essentially use a `pip install` to check for packages, which is a no-op if the package is already present. Should we change this for the other notebooks for consistency (they currently use a more complex `python -c import` pattern)?

e.g. DRISE, OcclusionSaliency, SimilarityScoring, SuperPixelSaliency, covid_classification

Reply:
We probably should, but I initially considered that out of scope for this PR. If we want to just "get it done" I can make that update, but it's just lower on the local priority list.

Reply:
Sounds good, I'll add it to the board as something to tackle in another PR.